### DISCUSSION TITLE
DateOnly and TimeOnly serialization

### DISCUSSION THREAD

**Discussion by Fabiano-Rezende** [MarimerLLC/csla#3704] [Ideas]
Are there plans to evolve MobileFormatter so that it serializes DateOnly and TimeOnly types?
Kind regards.

**Comment by rockfordlhotka** [comment]
Not specifically, though that's a good idea.

There are open issues for `DateTimeOffset` and also a more general solution to have the ability to serialize arbitrary types:

https://github.com/MarimerLLC/csla/issues/2531

**Reply by rockfordlhotka** [reply]
I've filed an issue for these types as well: #3711

**Reply by Chicagoan2016** [reply]
@rockfordlhotka Do TimeSpan properties get serialized? I have a few objects, derived from BusinessBase, that have TimeSpan properties which in turn correspond to 'time' columns in Sql Server.

**Reply by rockfordlhotka** [reply]
There's no current support for TimeSpan.
